Example/ViewController.swift

```diff
@@ -31,21 +31,22 @@ class ViewController: UIViewController {
         imagePicker.settings.selection.max = 5
         imagePicker.settings.theme.selectionStyle = .numbered
         imagePicker.settings.fetch.assets.supportedMediaTypes = [.image, .video]
-        imagePicker.settings.selection.unselectOnReachingMax = true
-
+       // imagePicker.settings.selection.unselectOnReachingMax = true
```

Comment:
Don't comment out code. Maybe create a new example that uses the delegate instead of closures.

Reply:
resolved